### DISCUSSION TITLE
animate-spin class not working in Angular 16

### DISCUSSION THREAD

**Discussion by piyushpranjal03** [tailwindlabs/tailwindcss#12305] [Help]
Hi Team,

I am using TailwindCss with Angular and I am not able to use this class `animate-spin` in a regular way.
Adding `animate-spin` to any element has no effect.
However when I am adding this keyframe in `style.css` file where we have three tailwind directives then it is working.

```
 @keyframes spin {
    from {
      transform: rotate(0deg);
    }
    to {
      transform: rotate(360deg);
    }
 }
```

**Versions Installed on my System**
Angular CLI: 16.1.6
Node: 18.16.0
Package Manager: npm 9.5.1
OS: win32 x64
tailwindcss: "^3.3.3"

**Comment by RobinMalfait** [comment]
Hey!

Please provide a minimal reproduction repo so that we can take a look. In the meantime, also take a look at the troubleshooting docs: https://tailwindcss.com/docs/content-configuration#troubleshooting

**Comment by puschie286** [comment]
transform breaks the animation if you have it on the same element. this can be fixed by explicit setting the 0 (from) statement in the animation - currently only the 100% (to) statement is exportet/set (tested with 3.3.5)
